src/Dialogs/DataField.cpp

```diff
@@ -64,21 +66,19 @@ EditDataFieldDialog(const TCHAR *caption, DataField &df,
     // signed or unsigned depends on min if value >= 0 or < 0...
     if (dfi.GetMin() >= 0) {
       unsigned value = dfi.GetValue(); // min is >= 0!
-      if (!NumberEntryDialog(caption, value,
-          log10(dfi.GetMax()) + 1))
+      if (!NumberEntryDialog(caption, value, log10(dfi.GetMax()) + 1))
         return false;
 
       dfi.ModifyValue(value); // SetAsInteger with unsigned!
       return true;
     } else {
       /* with signed range has to avoid the length of negative AND
-      * positiv numbers */
-      int value = dfi.GetValue();  // min is < 0!
+       * positiv numbers */
```

Comment:
If fixing the indent, can you also fix the spelling mistake? 

The two spaces after function name in former line 76 are intentional.